pkg/operator/ceph/object/objectstore.go

```diff
@@ -195,6 +195,12 @@ func deleteSingleSiteRealmAndPools(objContext *Context, spec cephv1.ObjectStoreS
 // This is used for quickly getting the name of the realm, zone group, and zone for an object-store to pass into a Context
 func getMultisiteForObjectStore(clusterdContext *clusterd.Context, spec *cephv1.ObjectStoreSpec, namespace, name string) (string, string, string, error) {
 	ctx := context.TODO()
+
+	// TODO: check or fix https://github.com/rook/rook/issues/6342
+	if spec.IsExternal() {
+		return "", "default", "default", nil
```

Comment:
Please comment on why we return "default" and what it means.

Reply:
Sure